server/test/christmas-tree-controllers.es6

```diff
@@ -0,0 +1,101 @@
+'use strict';
+
+const request = require('supertest');
+
+const server = require('./mock-aws-app.es6');
+
+const chai = require('chai');
+const expect = chai.expect;
+
+describe('christmas tree controller tests', () => {
+  describe('get forest info', () => {
+    it('should return a 200 response', done => {
+      request(server)
+        .get('/forests/1/regulations')
+        .set('Accept', 'application/json')
+        .expect('Content-Type', /json/)
+        .expect(200, done);
+    });
+
+    it('should include a field for species', done => {
+      request(server)
+        .get('/forests/1/regulations')
+        .set('Accept', 'application/json')
+        .expect('Content-Type', /json/)
+        .expect(function(res){
+          expect(typeof res.body.forest.species).to.not.equal('undefined');
```

Comment:
These are pretty vague expectations, let's make them more robust than `non undefined` all the way down.

Reply:
@lauraGgit we have added this task in a sprint-2 story

Reply:
@lpendem as a task to which story?

Reply:
https://github.com/nciinc/fs-intake-module/blob/sprint-2-development/frontend/e2e/xmas-tree-species.e2e-spec.ts

Reply:
@lauraGgit sorint-2 story https://github.com/nciinc/fs-intake-module/issues/5